nestjs/src/notifications/notifications.controller.ts

```diff
@@ -0,0 +1,48 @@
+import { Body, Controller, Delete, Get, Param, Post, Put, UseGuards } from '@nestjs/common';
+import { ApiForbiddenResponse, ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { DefaultGuard, RoleGuard } from 'src/auth';
+import { NotificationDto } from './dto/notification.dto';
+import { UpdateNotificationDto } from './dto/update-notification.dto';
+
+import { NotificationsService } from './notifications.service';
+
+@Controller('notifications')
+@ApiTags('notifications')
+@UseGuards(DefaultGuard, RoleGuard)
```

Comment:
you added RoleGuard, but I don't see any role decorator below

Reply:
e.g.

```
@RequiredAppRoles([Role.Admin])
```

Reply:
seems I have mistakenly read the `roleguard`. thanks - fixed